docs/_includes/intro.md

```diff
@@ -1,5 +1,8 @@
 The grounds that appear in the TesseLace index are the result of research by [Veronika Irvine](https://linktr.ee/tesselace/) and were generated using a [mathematical model](https://tesselacedotcom.wordpress.com/research/jmm2014/).  Some of these grounds are familiar, traditional grounds.  Others have not been found in known catalogues and are likely new.
 
+{% if include.skip_howto %}
```

Comment:
An optional how-to part in the import is a better solution than splitting. Could not find how to invert this if https://jekyllrb.com/docs/liquid/